CONTRIBUTING.md

```diff
@@ -0,0 +1,8 @@
+LightGBM has been developed and used by many active community members. Your help is very valuable to make it better for everyone.
+
+- Check for [Roadmap](https://github.com/microsoft/LightGBM/projects/1) and look for [issues with tag "help wanted"](https://github.com/microsoft/LightGBM/issues?q=is%3Aissue+is%3Aopen+label%3A%22help+wanted%22) and submit pull requests to address them. If you need development guideline, you can check the [Development-Guide](https://github.com/microsoft/LightGBM/blob/master/docs/Development-Guide.rst) or directly ask us in Issues/Pull Requests.
```

Comment:
As we have adopted the strategy of closing feature requests and practically have no open issues with `help wanted` tag, I don't think that it makes sense to ask users to look for them anymore.
```suggestion
- Check for the [Roadmap](https://github.com/microsoft/LightGBM/projects/1) and the [Feature Requests Hub](https://github.com/microsoft/LightGBM/issues/2302), and submit pull requests to address chosen issue. If you need development guideline, you can check the [Development Guide](https://github.com/microsoft/LightGBM/blob/master/docs/Development-Guide.rst) or directly ask us in Issues/Pull Requests.
```